src/tests/ftest/cart/util/cart_utils.py

```diff
@@ -224,6 +224,8 @@ def get_env(self):
 
         env += " -x CRT_ATTACH_INFO_PATH={!s}".format(daos_test_shared_dir)
 
+        env += " -x COVFILE=/tmp/test.cov"
```

Comment:
Should we consider using mkstemp for the location of code coverage ?

Reply:
I think this is ok for now. In ftests/scripts/setp_nodes.sh, we make sure there are no stale file laying around before getting the copy from the rpm install.

Reply:
I agree with @jolivier23 here.  Even though, today, we don't have issues with file name collisions such as this would produce, that doesn't mean we won't in the future (as testing evolves), and trying to track down such a collision issue will be quite difficult.  We should be defensive here.

Reply:
Do not disagree here. I've created a ticket DAOS-7163 to capture it.
Any reason this is not good enough for 1.2?